app/src/main/java/info/nightscout/android/medtronic/MainActivity.java

```diff
@@ -94,8 +95,12 @@
     private boolean hasZoomedChart = false;
 
     private NumberFormat sgvFormatter;
-    private boolean mmolxl;
-    private boolean mmolxlDecimals;
+    private static boolean mmolxl;
+    private static boolean mmolxlDecimals;
+
+    public static long timeLastGoodSGV = 0;
+    public static short pumpBattery = 0;
+    public static int countUnavailableSGV = 0;
```

Comment:
wait a sec

thinking of something else - need coffee - yeah those 3 were not changed to anything but statics